### PR TITLE
Added code and build files

### PR DESCRIPTION
**Pull request by psiinon**
Would this be better as a template repo (as suggested by @ricekot)?

I've copied across the files from https://github.com/zaproxy/zap-extensions/tree/main/addOns/simpleexample but as a result theres probably confusion as to whether this is a template, example, simple example .... but I've subitted the PR anyway so we can discuss that here..

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Rename the `yourname` directories to be your github name (or choose another package hierarchy which works better for you).
+
+1. Change the add-on ID `addonjava` to be a new ID which is not already on the [ZAP Marketplace](https://www.zaproxy.org/addons/).
+
+1. Change the code and help files to do whatever you want :smiley:
+
+1. Deploy and test your app locally, e.g. using `./gradlew copyZapAddon`
+
+1. Optional: Raise a [Pull Request](https://github.com/zaproxy/zaproxy/issues/new?assignees=&labels=marketplace&template=third-party-addon.yml) to get your add-on published on the [ZAP Marketplace](https://www.zaproxy.org/addons/).
```
Comment (kingthorin) [inline README.md:15]:
```suggestion
1. Optional: Raise a [Issue](https://github.com/zaproxy/zaproxy/issues/new?assignees=&labels=marketplace&template=third-party-addon.yml) to get your add-on published on the [ZAP Marketplace](https://www.zaproxy.org/addons/).
```

Comment (kingthorin) [reply README.md:15]: Should probably also add a bullet about scanners.md.

Comment (psiinon) [reply README.md]: Changed that :)
Not so sure about the scanners.md - I'd rather that was somewhere in https://www.zaproxy.org/docs/developer/ as I'd expect most new scan rules to go in existing add-ons...

Comment (kingthorin) [reply README.md]: Okay

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
:rocket: :tada:

**Comment by kingthorin** [comment]
Are we going to remove simple example from the other repo?

**Comment by psiinon** [comment]
Yeah, I'll do that, and update the dev docs...

**Pull request merged**